changelog/unreleased/issue-1734

```diff
@@ -1,16 +1,15 @@
-Enhancement: "Everything is ok" message after retrying
+Enhancement: Confirm successful retries after errors
```

Comment:
What about `Inform about successful retries after errors`? The meaning of "confirm" wasn't clear to me at first glance.

Reply:
Changed.